engine-comparison/begin-experiment.sh

```diff
@@ -34,28 +34,12 @@ done
 gcloud compute scp fengine-configs/ ${INSTANCE_NAME}:~/input --recurse
 rm -r fengine-configs
 
-# These will frequently/usually be defined by the user
-JOBS=${JOBS:-8}
-N_ITERATIONS=${N_ITERATIONS:-5}
+CONFIG=${SCRIPT_DIR}/config
+echo "BMARKS=$1" > ${CONFIG}/bmarks.cfg
```

Comment:
Yeah, this would just change the args for `begin-experiment`. It kind of feels like this is nicer to use, even if unifying to parameters.cfg is nicer to run